2021-05-05 09:00:04,778 | 17815 | proxy.proxy, open_connection, 242 | ERROR | Failed to connect to controller : [Errno 61] Connect call failed ('127.0.0.1', 6653)
2021-05-05 09:00:04,778 | 17815 | proxy.proxy, open_connection, 271 | DEBUG | reconnect to controller a few seconds later
2021-05-05 09:00:04,779 | 17815 | proxy.proxy, start_server, 132 | INFO | Server on ('127.0.0.1', 63333)
2021-05-05 09:00:04,779 | 17815 | proxy.proxy, start_server, 135 | INFO | Server serve forever ('127.0.0.1', 63333)
2021-05-05 09:00:09,780 | 17815 | proxy.proxy, open_connection, 242 | ERROR | Failed to connect to controller : [Errno 61] Connect call failed ('127.0.0.1', 6653)
2021-05-05 09:00:09,781 | 17815 | proxy.proxy, open_connection, 271 | DEBUG | reconnect to controller a few seconds later
2021-05-05 09:00:14,785 | 17815 | proxy.proxy, open_connection, 242 | ERROR | Failed to connect to controller : [Errno 61] Connect call failed ('127.0.0.1', 6653)
2021-05-05 09:00:14,786 | 17815 | proxy.proxy, open_connection, 271 | DEBUG | reconnect to controller a few seconds later
2021-05-05 09:00:19,791 | 17815 | proxy.proxy, open_connection, 242 | ERROR | Failed to connect to controller : [Errno 61] Connect call failed ('127.0.0.1', 6653)
2021-05-05 09:00:19,792 | 17815 | proxy.proxy, open_connection, 271 | DEBUG | reconnect to controller a few seconds later
2021-05-05 09:00:24,795 | 17815 | proxy.proxy, open_connection, 242 | ERROR | Failed to connect to controller : [Errno 61] Connect call failed ('127.0.0.1', 6653)
2021-05-05 09:00:24,795 | 17815 | proxy.proxy, open_connection, 271 | DEBUG | reconnect to controller a few seconds later
2021-05-05 09:00:29,797 | 17815 | proxy.proxy, open_connection, 242 | ERROR | Failed to connect to controller : [Errno 61] Connect call failed ('127.0.0.1', 6653)
2021-05-05 09:00:29,798 | 17815 | proxy.proxy, open_connection, 271 | DEBUG | reconnect to controller a few seconds later
2021-05-05 09:00:31,177 | 17815 | root, <module>, 53 | INFO | keyboardInterrupt : 
